src/prisma/_httpx_sync_http.py

```diff
@@ -0,0 +1,56 @@
+from typing import Any
+
+import httpx
+
+from ._types import Method
+from .http_abstract import AbstractResponse, AbstractHTTP
+
+
+__all__ = ('HTTP', 'Response', 'client')
+
+
+class HTTP(AbstractHTTP[httpx.Client, httpx.Response]):
+    # pylint: disable=invalid-overridden-method,attribute-defined-outside-init
+
+    session: httpx.Client
+
+    def download(self, url: str, dest: str) -> None:
+        with self.session.stream("GET", url, timeout=None) as resp:
```

Comment:
I prefer single quotes
```suggestion
        with self.session.stream('GET', url, timeout=None) as resp:
```